src/main/java/lotto/model/LottoResult.java

```diff
@@ -0,0 +1,54 @@
+package lotto.model;
+
+import lotto.model.wrapper.LottoMatchCount;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public enum LottoResult {
+    NONE(LottoMatchCount.of(0), 0L),
+    ONE(LottoMatchCount.of(1), 0L),
+    TWO(LottoMatchCount.of(2), 0L),
+    THREE(LottoMatchCount.of(3), 5000L),
+    FOUR(LottoMatchCount.of(4), 50000L),
+    FIVE(LottoMatchCount.of(5), 1500000L),
+    SIX(LottoMatchCount.of(6), 2000000000L),
```

Comment:
Java 7부터는 `100_000_000`처럼 숫자에 밑줄을 사용하여 가독성을 높일 수 있어요.

```suggestion
    SIX(LottoMatchCount.of(6), 2_000_000_000L),
```